resources/translations/lbr/fi.edn

```diff
@@ -0,0 +1,14 @@
+{:t
```

Comment:
These seem like part of the LBR deployment. If so, they should be in the rems-deploy repository IMO

Reply:
Yes, let's put them to the deploy repo.